packages/nc-gui/components/smartsheet-toolbar/FullScreen.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Component Syntax is a bit off here,
First `<script setup>` then `<template>` 

Doesn't eslint fix this for you?